### DISCUSSION TITLE
问问大佬前端主题切换如何实现？

### DISCUSSION THREAD

**Discussion by d1258zzz** [mirai-mamori/Sakurairo#439] [Q&A]
如题，希望制作一个主题切换的组件

**Comment by bymoye** [comment]
什么叫 主题切换

**Comment by d1258zzz** [comment]
[![](https://pic.imgdb.cn/item/621b1e862ab3f51d916bdf4d.jpg)](https://pic.imgdb.cn/item/621b1e862ab3f51d916bdf4d.jpg)
就这个？
具体问问如何实现？

**Reply by bymoye** [reply]
这只是切换背景和字体而已
